### PR TITLE
Added $response variable to onSent method and type casting.

### PR DESCRIPTION
**Pull request by aydinfatih**
Added $response variable to onSent method.
Added $channel variable type cast.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -42,12 +42,12 @@ class UserRegisteredNotification extends Notification
                     ->line('bar');
     }
     
-    public function onSending($channel): void
+    public function onSending(string $channel, $response): void
```
Comment (tkaratug) [inline README.md]: According to Laravel's NotificationSent event, `$response` must be nullable here.

File: tests/NotificationEventSubscriberTest.php [hunk 2]
```diff
@@ -36,12 +36,12 @@ public function toMail($notifiable): MailMessage
         return new MailMessage;
     }
 
-    public function onSent($channel): void
+    public function onSent(string $channel, $response): void
```
Comment (tkaratug) [inline tests/NotificationEventSubscriberTest.php]: `$response` must be nullable in here too.

Comment (aydinfatih) [reply tests/NotificationEventSubscriberTest.php]: The $response variable is set to nullable.

File: README.md (hunk 1 above)
Comment (aydinfatih) [reply README.md]: The $response variable is set to nullable.

**Pull request merged**